LifeCycleAssessment_Engine/Query/ProjectLifeCycleAssessmentResults.cs

```diff
@@ -0,0 +1,166 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.LifeCycleAssessment.Results;
+using BH.oM.Reflection.Attributes;
+using System;
+using System.ComponentModel;
+using System.Linq;
+
+namespace BH.Engine.LifeCycleAssessment
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****   Public Methods                          ****/
+        /***************************************************/
+
+        [Description("Query the standard project results from a complete ProjectLifeCycleAssessment evaluation for pushing to datasets.")]
+        [Input("lcaResult", "Supply a complete LifeCycleAssessmentResult to query.")]
+        [Output("projectResults", "The combined amount of kgCO2e of the objects provided.")]
+        public static ProjectLifeCycleAssessmentResult ProjectLifeCycleAssessmentResult(this LifeCycleAssessmentResult lcaResult)
+        {
+            double buildingLifespan = lcaResult.LifeCycleAssessmentScope.BuildingLifespan;
+            if (double.IsNaN(buildingLifespan))
+            {
+                BH.Engine.Reflection.Compute.RecordNote("No building lifespan has been provided for this entry. Returning zero.");
+                buildingLifespan = 0;
+            }
+
+            string constructionScope = "";
+            if (lcaResult.LifeCycleAssessmentScope.ConstructionScopeNew == true)
+                constructionScope = "New Construction";
+            else constructionScope = "Renovation";
+
+            string contact = lcaResult.LifeCycleAssessmentScope.ContactName;
+            if (contact == "")
+            {
+                BH.Engine.Reflection.Compute.RecordNote("Please enter your contact information within the LifeCycleAssessmentScope.");
+                contact = "Undefined";
+            }
+
+            string elementScope = "";
+            if(lcaResult.Results.Select(x => x.Scope).Count() > 0)
+            {
+                foreach (var b in lcaResult.Results.ToList())
+                    elementScope += b.Scope.ToString() + ",";
+                elementScope = elementScope.Remove(elementScope.LastIndexOf(','));
+            }
+            else
+            {
+                BH.Engine.Reflection.Compute.RecordWarning("No objects have been provided for evaluation within the discipline-specific scope objects.");
+            }
+
+            string gravitySys = System.Convert.ToString(lcaResult.LifeCycleAssessmentScope.GravityStructuralMaterial);
+            if(gravitySys == "Undefined")
```

Comment:
```suggestion
            if(gravitySys.ToLower() == "undefined")
```

If we have to do string matching, I'm guessing there's no difference between `Undefined`, `undefined`, and `uNdEfINed`? This way allows the user to be lazy with a `u` rather than forcing a `U`.

This is just a suggestion - happy to leave as is if the current implementation is more accurate :smile:

Reply:
Since the Enum value will always return an Upper `Undefined` I'm good to continue as implemented if ok!